transports/rabbitmq/index.md

```diff
@@ -14,6 +14,8 @@ tags:
 
 Provides support for sending messages over [RabbitMQ](http://www.rabbitmq.com/) using the [RabbitMQ .NET Client](https://www.nuget.org/packages/RabbitMQ.Client/).
 
+WARNING: The transport is not compatible with versions of the RabbitMQ broker earlier than 3.4.0.
```

Comment:
I prefer: 
> The transport is not compatible with RabbitMQ broker Versions 3.3.X and below.

Mostly because this is what we tested. It's the fact that we will not work with versions lower than 3.4.0.